RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
+Unreleased
```

Comment:
I noticed that there's no section for version `1.55.0`, probably this one is the one for that, wdyt?

Reply:
You are right. Updated with 9a14ab6291833f06c75532bdcacb8b664d08197b